packages/graphql-codegen-cli/src/utils/listr-renderer.ts

```diff
@@ -0,0 +1,62 @@
+import chalk from 'chalk';
+import * as logUpdate from 'log-update';
+import * as identString from 'indent-string';
+import * as UpdateRenderer from 'listr-update-renderer';
+import { DetailedError, isDetailedError } from '../errors';
+import { ListrTask } from 'listr';
+
+export class Renderer {
+  private updateRenderer: any;
+
+  constructor(tasks: ListrTask, options: any) {
+    this.updateRenderer = new UpdateRenderer(tasks, options);
```

Comment:
I could extend Renderer with UpdateRenderer but it throws an error... that's why I did it this way.